libvast_test/vast/test/fixtures/filesystem.hpp

```diff
@@ -26,7 +26,7 @@ struct filesystem {
       FAIL(vast::render(err));
   }
 
-  vast::path directory = "vast-unit-test";
+  const vast::path directory = "vast-unit-test";
```

Comment:
What motivated this change? I definitely agree with it, just curious what caused you to make it as part of this PR.

Reply:
Maybe I shouldn't have, but the initialization code in the fixture for the index unit test just threw me off with the modification to this variable.

Reply:
It's fine to have in here. Was just curious. :-)